drivers/income_benefits_report/app/models/income_benefits_report/client.rb

```diff
@@ -0,0 +1,43 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# NOTE: This report runs all calculations against the most-recently started enrollment
+# that matches the filter scope for a given client
+module IncomeBenefitsReport
+  class Client < GrdaWarehouseBase
+    self.table_name = 'income_benefits_report_clients'
+    belongs_to :report
```

Comment:
Does this need a `class_name`?

Reply:
Shouldn't since it's in the same module and the name matches up with the relation name.